package.json

```diff
@@ -67,24 +67,24 @@
     "babylon": "^6.18.0",
     "chai": "^4.2.0",
     "coveralls": "^3.0.6",
-    "cross-env": "^4.0.0",
+    "cross-env": "^7.0.2",
     "eslint": "^2 || ^3 || ^4 || ^5 || ^6 || ^7.0.0-0",
     "eslint-import-resolver-node": "file:./resolvers/node",
-    "eslint-import-resolver-typescript": "^1.0.2",
+    "eslint-import-resolver-typescript": "^2.0.0",
     "eslint-import-resolver-webpack": "file:./resolvers/webpack",
     "eslint-import-test-order-redirect": "file:./tests/files/order-redirect",
     "eslint-module-utils": "file:./utils",
     "eslint-plugin-eslint-plugin": "^2.2.1",
     "eslint-plugin-import": "2.x",
     "in-publish": "^2.0.0",
     "linklocal": "^2.8.2",
-    "mocha": "^3.5.3",
-    "nyc": "^11.9.0",
+    "mocha": "^7.2.0",
```

Comment:
mocha > 3 drops support for older engines, so we'll never be able to upgrade beyond mocha v3.